protocols/relay/src/behaviour.rs

```diff
@@ -54,6 +54,8 @@ pub struct Config {
     pub reservation_duration: Duration,
     pub reservation_rate_limiters: Vec<Box<dyn rate_limiter::RateLimiter>>,
 
+    pub substream_timeout: Duration,
+    pub max_concurrent_streams: usize,
```

Comment:
```suggestion
    pub max_concurrent_streams_per_connection: usize,
```

Would be more descriptive, right?